src/release/mergeHelper.d

```diff
@@ -195,7 +195,15 @@ class PatchMerger
 
             foreach (pending; local_pending_branches)
             {
-                this.tagAndMerge(pending.branch, pending.merged);
+                try this.tagAndMerge(pending.branch, pending.merged);
+                catch (Exception exc)
+                {
+                    import std.stdio;
+                    import colorize;
+                    writefln("Failed to handle branch %s: %s".color(fg.red),
```

Comment:
I think it will be less noisy to just `exc.msg = "When handling branch " ~ pending ~ ":\n" ~ exc.msg` and rethrow.